src/index.ts

```diff
@@ -0,0 +1,566 @@
+export { Client } from './client'
```

Comment:
the file is kind of long, so I think it will be nice reorganize and separate with some highlight comments
```typescript
/*** here json rpc answer ***/
....
/*** here utils types alias ***/
...
/*** .... ***/
```
also one question:
you are using only type alias instead of interface, do you want only object literal?

Reply:
Reorganized. I put `ClientOption` in `Client.ts` as well as I thought it makes more sense.

Reply:
thanks :)